picard/ui/coverartbox.py

```diff
@@ -394,19 +403,54 @@ def load_remote_image(self, url, mime, data):
         elif isinstance(self.item, Track):
             track = self.item
             track.album.enable_update_metadata_images(False)
-            track.metadata.set_front_image(coverartimage)
+            drop_image(track)
             track.metadata_images_changed.emit()
             for file in track.iterfiles():
-                file.metadata.set_front_image(coverartimage)
+                drop_image(file)
                 file.metadata_images_changed.emit()
                 file.update()
             track.album.enable_update_metadata_images(True)
             track.album.update_metadata_images()
             track.album.update(False)
         elif isinstance(self.item, File):
             file = self.item
-            file.metadata.set_front_image(coverartimage)
+            drop_image(file)
             file.metadata_images_changed.emit()
             file.update()
         self.cover_art.set_metadata(self.item.metadata)
         self.show()
+
+    def setBehaviourOnImageDrop(self, behaviour):
```

Comment:
That function is a slot called by the Qt signal-slot mechanism, so I thought it was ok to use camelCase, but ok, I'll change that.